<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui</class>
 <widget class="QMainWindow" name="ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>情绪</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>动作</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>意境</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget_top">
          <column>
           <property name="text">
            <string>情绪/神情/动作</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>正面情绪</string>
           </property>
           <item>
            <property name="text">
             <string>爱意、爱慕</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>满足</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>欣快</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>感动感恩</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>快乐</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>幸福</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自信</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>骄傲</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>愉快</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>中性情绪</string>
           </property>
           <item>
            <property name="text">
             <string>好奇</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>同情心</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>羡慕</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>惊讶</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>震惊</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>同情</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>出世</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>负面情绪</string>
           </property>
           <item>
            <property name="text">
             <string>恐惧</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>内疚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>惊悚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>愤怒</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>烦恼</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>丧恸</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>焦虑</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷漠</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>敬畏</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>厌烦</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>轻蔑鄙视</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>抑郁</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>尴尬</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>失败</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>欲望</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>失望</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>仇恨</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>羞辱</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>忌妒</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>孤独</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>情欲</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>后悔</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自责</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>悲伤</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>担心、担忧或忧虑</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>痛苦</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>羞怯、害羞</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>羞耻</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>怅然若失</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>神情</string>
           </property>
           <item>
            <property name="text">
             <string>眼里有光</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>动作</string>
           </property>
           <item>
            <property name="text">
             <string>怔了怔</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>意境</string>
           </property>
           <item>
            <property name="text">
             <string>一片安静</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="bookList">
          <column>
           <property name="text">
            <string>书列表</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>择天记</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>诛仙</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>搜神记</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>名词解释</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textEdit1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>363</width>
            <height>390</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1400</width>
            <height>390</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit2"/>
        </item>
        <item>
         <widget class="QPushButton" name="insertWordBtn">
          <property name="text">
           <string>插入关键词</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchBtn1">
            <property name="text">
             <string>检索词库</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveWordBtn">
            <property name="text">
             <string>保存名词</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveTextBtn">
            <property name="text">
             <string>录入句库</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textEdit2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>363</width>
            <height>390</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEdit3">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>819</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>1400</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
